tidy3d/components/device/simulation.py

```diff
@@ -200,7 +236,9 @@ def check_zero_dim_domain(cls, val, values):
         num_zero_dims = np.sum(zero_dimensions)
 
         if num_zero_dims > 1:
-            mssg = f"Your current HeatSimulation has zero size along the {zero_dim_str}dimensions. "
+            mssg = (
+                f"Your current DeviceSimulation has zero size along the {zero_dim_str}dimensions. "
```

Comment:
variable / class names should go in single quotes `'` (a few other instances below, double check)`.

`along the {zero_dim_str}dimensions` should there be a space after `}`? double check

Reply:
a space is added to `zero_dim_str` above